lib/board.rb

```diff
@@ -3,56 +3,40 @@ class Board
 
   def initialize 
     @cells = {
-      one: ' ', two: ' ', three: ' ',
-      four: ' ', five: ' ', six: ' ',
-      seven: ' ', eight: ' ', nine: ' ',
+      one: '', two: '', three: '',
+      four: '', five: '', six: '',
+      seven: '', eight: '', nine: '',
     }
   end
 
-  def cell_empty?(selected_cell)
-    return true if @cells[selected_cell.to_sym] == ' '
-    false
+  def cell_empty?(cell)
+    (@cells[cell]).empty?
   end
 
-  def game_won?(piece)
-    return true if win?(board_rows, piece) || win?(board_columns, piece) || win?(board_diagonals, piece)
-  end
-
-  def render
-    puts "#{@cells[:one]}|#{@cells[:two]}|#{@cells[:three]}"
-    puts '-----'
-    puts "#{@cells[:four]}|#{@cells[:five]}|#{@cells[:six]}"
-    puts '-----'
-    puts "#{@cells[:seven]}|#{@cells[:eight]}|#{@cells[:nine]}"
-  end
-
-  def win?(win_type, piece)
-    win_type.each_slice(3) do |type|
-      return true if type.all? {|cell| cell == piece }
-    end
-    false
+  def set_cell(cell, piece)
+      @cells[cell] = piece
   end
 
-    def board_rows
-    rows = [
-      @cells[:one], @cells[:two], @cells[:three],
-      @cells[:four], @cells[:five], @cells[:six],
-      @cells[:seven], @cells[:eight], @cells[:nine]
-    ]
+  def game_won?(piece)
+    return true if win?(piece)
   end
 
-  def board_columns
-    columns = [
-      @cells[:one], @cells[:four], @cells[:seven],
-      @cells[:two], @cells[:five], @cells[:eight],
-      @cells[:three], @cells[:six], @cells[:nine]
-    ]
+  def win_helper(x, y, z)
+    !(x).empty? && x == y && y == z
```

Comment:
it's not a hugely important change, but it is good to know how to use bits and pieces from Rails if you need to. I'm curious what error you received when trying to include ActiveSupport?